tests/runner.py

```diff
@@ -58,6 +58,23 @@ class Result:
     expected: str
     command: list[str]
 
+def unordered_compare(out, expected):
+    if out == expected:
+        return True
+
+    sorted_out = out.split()
+    sorted_expected = expected.split()
+    sorted_out.sort()
+    sorted_expected.sort()
+
+    if len(sorted_out) != len(sorted_expected):
+        return False
+
+    for i in range(len(sorted_out)):
+        if sorted_out[i] != sorted_expected[i]:
+            return False
+
+    return True
```

Comment:
```suggestion
    out_parts = out.split()
    expected_parts = expected.split()
    return sorted(out_parts) == sorted(expected_parts)
```

Does this work just as well? We could use `set` instead of `sorted` too, but that wouldn't detect the difference between `foo bar` and `foo foo foo bar`.

Reply:
That's a good idea, let me try it.